src/figuresClasses.ts

```diff
@@ -1,19 +1,99 @@
-export interface Figure {
+function isTrianglePossible(a: number, b: number, c: number): boolean {
+  return a + b > c && a + c > b && b + c > a;
+}
+
+function roundDownFloat(number: number, amountOfDecimals: number = 2): number {
+  const numbersAfterFloat = 10 ** amountOfDecimals;
+
+  return Math.floor(number * numbersAfterFloat) / numbersAfterFloat;
+}
 
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
 }
 
-export class Triangle {
+type Color = 'red'|'blue'|'green';
```

Comment:
What is the reason for using enums and unions together?

Reply:
If I use enum tests go well but they are showing problems
<img width="420" alt="image" src="https://github.com/mate-academy/ts_figures_classes/assets/90126198/3d754f3e-c47b-47cf-87bc-380b9b0390d1">